contents/handbook/product/user-feedback.md

```diff
@@ -0,0 +1,76 @@
+---
+title: User feedback
+sidebar: Handbook
+showTitle: true
+---
+
+Gathering user feedback is critical for the development of our product. It's not about casually listening to users, or just paying attention to bug reports, we actively seek (outbound) input in everything we work on. In addition to having multiple channels to continuously receive inbound feedback, we generally do active outbound feedback requests for:
+- General product and experience feedback. Continuous effort to gather general feedback on the product and their holistic experience with PostHog. Usually run by the [Feedback Hero](#feedback-hero).
+- Usability tests. We generally run this for new big features the Engineering team is working on. Usually run by the Product Team.
+
+> 😍 Want to share feedback? Simply send us an email to [hey@posthog.com](mailto:hey@posthog.com) or reach out on [Slack](/slack). **We're always happy to hear from you!**
+
+
+### Feedback Hero
+Previously a single person owned the feedback process. As we grew, we created the Feedback Hero ([see discussion](https://github.com/PostHog/product-internal/issues/98)) to scale our feedback process. The Feedback Hero is **rotated every two weeks**. This person is in charge of:
+1. Proactive reach out to users to schedule feedback calls.
+2. Having the feedback calls.
+3. Housekeeping. Logging the relevant notes and other reports (see [process](#feedback-process)) below. Send the user applicable rewards as a thank you for their time.
```

Comment:
Do we want to document what the rewards are, or the process for this?

Reply:
I've added some details on this but in general, if it doesn't fall on the typical $50 merch code, it's best left to the interviewee's best judgement, there's a multitude of factors to consider.